apim/3.x/README.adoc

```diff
@@ -163,6 +187,38 @@ See
 https://github.com/helm/charts/tree/master/stable/mongodb-replicaset[MongoDB
 replicaset] for detailed documentation on helm chart.
 
+==== PostgresSQL (via JDBC Connection) 
+To install a new PostgresSQL database, use the command below and update the `username`, `password`, and `databasename` parameters: 
+----
+helm install --set postgresqlUsername=postgres --set postgresqlPassword=P@ssw0rd 
+--set postgresqlDatabase=graviteeapim postgres-apim bitnami/postgresql
+----
+
+NOTE: When you install APIM, it automatically uses the default value for the host and runs on `apim.example.com`. To use a specific host name, search through our documentation and replace any values in the `values.yml` configuration file that use the default value of `apim.example.com`. 
```

Comment:
Nice note :+1: 
But I don't think it's related to PosgtreSQL configuration.
Maybe we can move it in another section ? For example the "Helm Install Command Tips" ?